### PR TITLE
Set up the new configuration for CircleCI

### PR DESCRIPTION
**Pull request by chubchenko**

### CODE REVIEW

**Comment by djezzzl** [comment]
Can we also add a cron job to be running once per week for master only, please?

**Comment by chubchenko** [comment]
> Can we also add a cron job to be running once per week for master only, please?

Yes, sure. I will add it when the build will be green.

**Comment by djezzzl** [comment]
Hi @chubchenko ,

First of all, thank you for your contribution! 

Could you please say if we can merge this nice improvements?

File: gemfiles/ar_7_0.gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'activerecord', '~> 7.0.0.alpha2'
```
Comment (djezzzl) [inline gemfiles/ar_7_0.gemfile:5]: Can we point to master instead? What do you think?

**Pull request merged**